example/example-command/example-command-handler/src/main/java/uk/gov/justice/services/example/cakeshop/command/handler/MakeCakeCommandHandler.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.services.example.cakeshop.command.handler;
+
+import org.slf4j.Logger;
+import uk.gov.justice.services.core.annotation.Handles;
+import uk.gov.justice.services.core.annotation.ServiceComponent;
+import uk.gov.justice.services.eventsourcing.source.core.exception.EventStreamException;
+import uk.gov.justice.services.messaging.Envelope;
+
+import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.justice.services.core.annotation.Component.COMMAND_HANDLER;
+
+@ServiceComponent(COMMAND_HANDLER)
+public class MakeCakeCommandHandler {
+
+    private static final Logger LOGGER = getLogger(MakeCakeCommandHandler.class);
+
+    @Handles("cakeshop.commands.make-cake")
+    public void handle(final Envelope envelope) throws EventStreamException {
+        LOGGER.info("=============> Inside make-cake Command Handler");
```

Comment:
We haven't really implemented make-cake yet. Only add-recipe. Handlers have been created just to ensure it doesn't crash.